fcrepo-configs/src/main/java/org/fcrepo/config/OcflPropsConfig.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.fcrepo.config;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.PostConstruct;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * Fedora's OCFL related configuration properties
+ */
+@Configuration
+public class OcflPropsConfig {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(OcflPropsConfig.class);
+
+    private static final String OCFL_STAGING = "staging";
+    private static final String OCFL_ROOT = "ocfl-root";
+    private static final String OCFL_TEMP = "ocfl-temp";
+
+    @Value("${fcrepo.ocfl.staging:#{fedoraPropsConfig.fedoraData.resolve('" + OCFL_STAGING + "')}}")
+    private Path fedoraOcflStaging;
+
+    @Value("${fcrepo.ocfl.root:#{fedoraPropsConfig.fedoraData.resolve('" + OCFL_ROOT + "')}}")
+    private Path ocflRepoRoot;
+
+    @Value("${fcrepo.ocfl.temp:#{fedoraPropsConfig.fedoraData.resolve('" + OCFL_TEMP + "')}}")
+    private Path ocflTemp;
+
+    @PostConstruct
+    private void postConstruct() throws IOException {
+        LOGGER.debug("Fedora staging: {}", fedoraOcflStaging);
```

Comment:
Normally I do not like the logs to be too chatty... but these three messages are important to see at startup.
Can you change them to `LOGGER.info()`?

Reply:
done